Flow.Launcher.Infrastructure/Hotkey/HotkeyModel.cs

```diff
@@ -137,8 +137,13 @@ public override string ToString()
             }
             return string.Join(" + ", keys);
         }
-
-        public bool Validate()
+        
+        /// <summary>
+        /// Validate hotkey
+        /// </summary>
+        /// <param name="validateKeyGestrue">Try to validate hotkey as a KeyGesture.</param>
+        /// <returns></returns>
+        public bool Validate(bool validateKeyGestrue = false)
```

Comment:
Do we need this option? Shouldn't we always validate?

If needed can we set validate to true by default?